release-common.nix

```
@@ -70,7 +70,8 @@ rec {
           sha256 = "1pij0v449p166f9l29x7ppzk8j7g9k9mp15ilh5qxp29c7fnvxy2";
         }) ];
         */
-      }));
+      }))
+    ++ lib.optional (stdenv.isLinux || stdenv.isDarwin) google-cloud-cpp;
```

so that it keeps working on nixox-19.03:
```suggestion
    ++ lib.optional (pkgs ? google-cloud-cpp && (stdenv.isLinux || stdenv.isDarwin)) google-cloud-cpp;
```

---

Thanks! Fixed.